teacher.rb

```diff
@@ -0,0 +1,13 @@
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(id, age, parent_permision, specialization, name = 'unknown')
+    super(id, name, age, parent_permision)
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=parent_permission%20as%20parameter.-,name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true.,-Getters%20for%20%40id), The `parent_permission` parameter has a default value `true`. Please add it.